packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1112,6 +1112,10 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
 		return this.indexes.delete(index);
 	}
 
+	public hasIndex(index: IntervalIndex<TInterval>): boolean {
```

Comment:
no need to add this API--you can just use the return value of `removeIndex`.

Reply:
ah yes we can simply reordered the logic. I initially wanted to remove the index after clearing all intervals within it, but it was not necessary.